src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,49 @@
+package core.basesyntax;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.model.Operation;
+import core.basesyntax.service.FormaterService;
+import core.basesyntax.service.ReaderService;
+import core.basesyntax.service.ReportService;
+import core.basesyntax.service.WriterService;
+import core.basesyntax.service.impl.CsvReaderServiceImpl;
+import core.basesyntax.service.impl.FormaterServiceImpl;
+import core.basesyntax.service.impl.FruitStorageServiceImpl;
+import core.basesyntax.service.impl.ReportServiceImpl;
+import core.basesyntax.service.impl.WriterServiceImpl;
+import core.basesyntax.strategy.OperationHandler;
+import core.basesyntax.strategy.impl.BalanceOperationHandler;
+import core.basesyntax.strategy.impl.PurchaseOperationHandler;
+import core.basesyntax.strategy.impl.ReturnOperationHandler;
+import core.basesyntax.strategy.impl.SupplyOperationHandler;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class Main {
+    public static final String PATH_TO_INPUT_FILE = "src/main/java/resources/input.csv";
+    public static final String PATH_TO_REPORT_FILE = "src/main/java/resources/report.csv";
+
+    public static final Map<Operation, OperationHandler> HANDLER_MAP = new HashMap<>();
+
+    static {
+        HANDLER_MAP.put(Operation.BALANCE, new BalanceOperationHandler());
+        HANDLER_MAP.put(Operation.PURCHASE, new PurchaseOperationHandler());
+        HANDLER_MAP.put(Operation.RETURN, new ReturnOperationHandler());
+        HANDLER_MAP.put(Operation.SUPPLY, new SupplyOperationHandler());
+    }

```

Comment:
I think you don't need this class field. It is better to create a private method that will return such a Map when needed